core/src/main/java/io/questdb/cairo/CairoException.java

```diff
@@ -331,7 +324,6 @@ protected void clear(int errno) {
         cacheable = false;
         interruption = false;
         authorizationError = false;
-        entityDisabled = false;
```

Comment:
I think I'm lacking context here, but why is the field removed?

Reply:
> I think I'm lacking context here, but why is the field removed?

Not needed anymore.
The piece code, which used `CairoException::isEntityDisabled()` has been removed.